conf/application.conf

```diff
@@ -67,9 +67,8 @@ microservice {
     }
 
     contact-frontend {
-      protocol = http
-      host = localhost
-      port = 9250
+      external-url = "http://localhost:9250"
+      serviceId = "AOSS"
```

Comment:
there is already `contact-frontend.serviceId = "AOSS"` lower down if you could remove that line

Reply:
I meant to remove that, thx